src/apprt/gtk/Surface.zig

```diff
@@ -840,6 +840,12 @@ pub fn setInitialWindowSize(self: *const Surface, width: u32, height: u32) !void
     );
 }
 
+pub fn setInitialWindowPosition(self: *const Surface, x: i32, y: i32) !void {
+    // We need the surface's window to set the position.
+    const window = self.container.window() orelse return;
+    c.gtk_window_move(@ptrCast(window.window), x, y);
```

Comment:
`gtk_window_move` is not a valid GTK4 API. If this is PR is accepted for macOS, `setInitialWindowPosition` should be a no-op on GTK.

Reply:
yup! I made a note of that in the "Work in progress" to-do list. I'd personally like to explore if there are any compatibility workarounds for GTK4 so that this can still be an option but I don't currently have the time today to dig deeper into it so for now this is just a draft. If I don't end up finding any "good" way to do this, I'll be sure to make it clear that this config option only works on macOS and X11 (IIRC there is a build config option to build using GTK3 for X11)

Reply:
The GTK devs are pretty adamant that positioning windows in that way is out of scope for GTK. Ghostty will not build against GTK3. GTK4 can use X11 but even in that case you can't position windows. It's possible that there's a Wayland protocol to do that but I don't think that we're ready to be talking to Wayland directly. And compositor specific solutions I think are not a viable option either (there's just too many of them).

Reply:
yeah I actually just did some digging around for a bit and nothing really seemed that great to me either, I removed the GTK implementation of `setInitialWindowPosition` in [826bc18](https://github.com/ghostty-org/ghostty/pull/3929/commits/826bc18204ae1aa580340f9aa8ef96e565de5e15) and updated the config docs appropriately